docs/get-started/create-a-project.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_label: 'Create a project'
+sidebar_position: 2
+---
+
+# Create a project
+
+First, create a project, move to it and initialize Git.
+```shell
+$ git init
+
+Initialized empty Git repository
+```
+
+After Git we also initialize DVC.
+```shell
+$ dvc init
+
+Initialized DVC repository.
+
+You can now commit the changes to git.
+```
+
+Then, we need to initialize a new Holium project on our file system with the command provisioned by
+the CLI.
+
+```shell
+$ holium init
+To track changes in the Holium project, run :
+
+        dvc add .holium/interplanetary
+        git add .holium
+
+Initialized Holium repository.
+```
+
+This command will have created a `.holium` folder in it that contains all the files related to your 
```

Comment:
Looks like we may have three broken internal links in total:
```
warn Docs markdown link couldn't be resolved: (get-started/install.md) in ./docs/home.md for version current
warn Docs markdown link couldn't be resolved: (get-started/quickstart.md) in ./docs/guides.md for version current
warn Docs markdown link couldn't be resolved: (../reference/cli/project-structure.md) in ./docs/get-started/create-a-project.md for version current
```

Reply:
Fixed this in https://github.com/polyphene/docs.holium.org/pull/16/commits/c4ed9968902ea13e74a06e8e00533f8f684ea1e1

Reply:
Still facing these ones:

```
warn Docs markdown link couldn't be resolved: (get-started/install.md) in /home/philippe/devpt/polyphene/product/holium-docs/docs/home.md for version current
warn Docs markdown link couldn't be resolved: (get-started/quickstart.md) in /home/philippe/devpt/polyphene/product/holium-docs/docs/guides.md for version current
```